protonvpn_linux_gui/protonvpn_gui.py

```diff
@@ -84,6 +84,10 @@ def init():
 
         if not os.path.isfile(GUI_CONFIG_FILE):
             initialize_gui_config()
+        
+        cli_log_path = os.path.join(CONFIG_DIR, "pvpn-cli.log")
+        if os.path.isfile(cli_log_path):
+            change_file_owner(cli_log_path)
```

Comment:
Why is that?

Reply:
Because technically the CLI is never launched and the log file is protected by root.